integration_tests/sdk/aqueduct_tests/error_handling_test.py

```diff
@@ -27,6 +32,15 @@ def test_handle_bad_op_error(client, data_integration):
         assert TIP_OP_EXECUTION in e.message
 
 
+def test_handle_bad_op_with_multiple_outputs(client, data_integration):
+    table_artifact = extract(data_integration, DataObject.SENTIMENT)
+
+    try:
```

Comment:
The way to do it in tests is to use `with pytest.raises(AqueductError, match=TIP_OP_EXECUTION)`